shard-operator/src/main/java/com/redhat/service/bridge/shard/operator/ManagerSyncServiceImpl.java

```diff
@@ -90,10 +90,9 @@ void syncUpdatesFromManager() {
                                     }
                                     if (y.getStatus().equals(ManagedResourceStatus.DEPROVISION)) { // Bridges to delete
                                         y.setStatus(ManagedResourceStatus.DELETING);
-                                        bridgeIngressService.deleteBridgeIngress(y);
                                         return notifyBridgeStatusChange(y)
                                                 .subscribe().with(
-                                                        success -> LOGGER.debug("Delete notification for Bridge '{}' has been sent to the manager successfully", y.getId()),
+                                                        success -> bridgeIngressService.deleteBridgeIngress(y),
```

Comment:
Is it not nice to also preserve the logging somewhere?

- But I guess no logging is consistent with the creation process.

Reply:
done